python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/conftest.py

```diff
@@ -189,6 +189,8 @@ def instance_dont_use_current_task(instance_cm, subnet):
             },
         }
     ) as dagster_instance:
+        # Not running in an ECS task
+        monkeypatch.setenv("ECS_CONTAINER_METADATA_URI_V4", None)
```

Comment:
So have we severed the need to launch tasks _from_ ECS. As in you can use the ECS launcher (when properly configured) when otherwise deployed to K8s, Docker, etc.?

Reply:
That's right - that happened a while ago